build/docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: '3.7'
+
+services:
+  # loki:
+  #   container_name: loki
+  #   image: grafana/loki:2.6.1
+  #   expose:
+  #     - 3100
+  #   ports:
+  #     - "3100:3100"
+  #   restart: unless-stopped
+  #   volumes:
+  #     - loki-data:/loki
+
+  # promtail:
+  #   container_name: promtail
+  #   image: grafana/promtail:2.6.1
+  #   volumes:
+  #     # custom config will read logs from the containers of
+  #     # this project
+  #     - ${PWD}/promtail:/etc/promtail
+  #     # to read container labels and logs
+  #     - /var/run/docker.sock:/var/run/docker.sock
+  #     - /var/lib/docker/containers:/var/lib/docker/containers
```

Comment:
either provide a comment explaining or remove this

Reply:
Will be removed